releases/release-1.34/README.md

```diff
@@ -0,0 +1,110 @@
+---
+title: "Kubernetes v1.34 Release Information"
+weight: 98
+slug: "release"
+aliases: [ "/release" ]
+description: |
+  Information regarding the current release cycle including important dates,
+  Release Team contact information, tracking spreadsheets and more!
+---
+
+# Kubernetes v1.34
+
+#### Links
+
+* [This document](https://git.k8s.io/sig-release/releases/release-1.34/README.md)
+* [Release Team](https://github.com/kubernetes/sig-release/blob/master/releases/release-1.34/release-team.md)
+* [Meeting Minutes](https://bit.ly/k8s134-releasemtg) (members of [release-team@] receive meeting invites)
+* [v1.34 Release Calendar](https://bit.ly/k8s-release-cal)
+* Contact: [#sig-release](https://kubernetes.slack.com/archives/C2C40FMNF) on
+  slack, [release-team](mailto://release-team@kubernetes.io) on e-mail
+* [Internal Contact Info](https://bit.ly/k8s134-contacts) (accessible only to members of [release-team@])
+
+#### Tracking docs
+
+* [Enhancements Tracking Board](https://bit.ly/k8s134-enhancements)
+* [Feature blog Tracking Board](https://bit.ly/k8s134-feature-blogs)
+* [Bug Triage Tracking Board](https://bit.ly/k8s134-bugtriage)
+* [CI Signal Project Board & Issue Tracking](https://bit.ly/k8s134-cisignal)
+* [Retrospective Document](https://bit.ly/k8s134-retro)
+* [kubernetes/kubernetes v1.34 milestone](https://github.com/kubernetes/kubernetes/milestone/67)
+
+#### Guides
+
+* [Targeting Issues and PRs to This Milestone](https://git.k8s.io/community/contributors/devel/sig-release/release.md)
+* [Triaging and Escalating Test Failures](https://git.k8s.io/community/contributors/devel/sig-testing/testing.md#troubleshooting-a-failure)
+
+## Summary
+
+The v1.34 release cycle is as follows:
+
+- **Monday 19th May 2025**: Week 1 — Release cycle begins
+- **Thursday 12th June 2025**: Week
+  4 — [Production Readiness Freeze][Production Readiness Freeze]
+- **[02:00 UTC Friday 20th June 2025 / 19:00 PST Thursday 19th June 2025](https://everytimezone.com/s/37be6888)**:
```

Comment:
Not sure if you have already considered this, but Thursday 19th June is a public holiday and a nonworking day for many people in the US

Reply:
I'm not sure if we want to move this deadline for one holiday, but if we do how about we shift this deadline to: `21:00 UTC Friday 20th June 2025 / 14:00 PST Friday 20th June 2025`?

Reply:
sgtm. Up to you and the release team of course (I don't mean to create a big fuss), just wanted to flag it

Reply:
> 21:00 UTC Friday 20th June 2025 / 14:00 PST Friday 20th June 2025

this time sounds good to me

Reply:
I agree with avoiding public holidays where we can. If we can move it to the day before (so Wednesday in this case), we might capture more attention from people who may end up taking Thursday-Friday-Saturday-Sunday as a four-day weekend. (Friday in this case avoids the Thursday holiday in the US but puts the deadline on the weekend for much of the rest of the world, so that worries me a bit.)

Reply:
good point about the weekend. would it work if we shift to the day before, keeping the original times?
`02:00 UTC Thursday 19th June 2025 / 19:00 PST Wednesday 18th June 2025`

and in that case, should the PRR freeze also be shifted a day before, to give a full week before enhancements freeze? (or does the one day difference not matter too much?)

Reply:
Yeah, I'm concerned about shortening the deadline for a single day holiday in the US. I think it's fairly fine to assume that if we keep it on Friday, KEP owners can get their SIGs to review it till Wednesday, if the reviewers won't be available. Shortening it for the rest of the world worries me as well.

Reply:
I have moved it to `21:00 UTC Friday 20th June 2025 / 14:00 PST Friday 20th June 2025` to accommodate for the US holiday.